mobile/application-manager-base.ts

```diff
@@ -69,5 +88,5 @@ export abstract class ApplicationManagerBase extends EventEmitter implements Mob
 	public abstract stopApplication(appIdentifier: string): IFuture<void>;
 	public abstract getInstalledApplications(): IFuture<string[]>;
 	public abstract canStartApplication(): boolean;
-	public abstract getApplicationsLiveSyncSupportedStatus(newApplications: string[]): IFuture<void>;
+	public abstract getIsLiveSyncSupportedForApplicationFromDevice(appIdentifier: string): IFuture<boolean>;
```

Comment:
this could be `protected` if it grunts, and it's name can be reconsidered
maybe something like `isLiveSyncSupportedOnDevice`